### PR TITLE
Feature/contact

### PR DESCRIPTION
**Pull request by Tougee**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -420,6 +420,7 @@
     <string name="document">文件</string>
     <string name="gallery">照片</string>
     <string name="video">视频</string>
+    <string name="contact">l联系人</string>
```
Comment (crossle) [inline app/src/main/res/values-zh-rCN/strings.xml]: Translation error

File: app/src/main/java/one/mixin/android/ui/conversation/FriendsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+class FriendsFragment : BaseFragment(), FriendAdapter.FriendListener {
+
+    companion object {
+        const val TAG = "FriendsFragment"
+        const val ARGS_IS_GROUP = "args_is_group"
+        const val ARGS_IS_BOT = "args_is_bot"
+
+        fun newInstance(conversationId: String, isGroup: Boolean, isBot: Boolean) = FriendsFragment().apply {
+            arguments = Bundle().apply {
```
Comment (crossle) [inline app/src/main/java/one/mixin/android/ui/conversation/FriendsFragment.kt]: Use bundleOf() or Fragment.withArgs instance fragment

File: app/src/main/java/one/mixin/android/ui/conversation/FriendsFragment.kt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                    }
+                }
+                adapter.friends = us
+            }
+        })
+    }
+
+    override fun onFriendClick(user: User) {
+        val fw = ForwardMessage(ForwardCategory.CONTACT.name, sharedUserId = user.userId)
+        ConversationActivity.show(context!!, conversationId, null, messages = arrayListOf(fw), isGroup = isGroup, isBot = isBot)
```
Comment (crossle) [inline app/src/main/java/one/mixin/android/ui/conversation/FriendsFragment.kt]: Dont use !! unpack the optional context, use requireContext() get the NonNull context

File: app/src/main/res/values-zh-rCN/strings.xml (hunk 1 above)
Comment (Tougee) [reply app/src/main/res/values-zh-rCN/strings.xml]: fixed

File: app/src/main/java/one/mixin/android/ui/conversation/FriendsFragment.kt (hunk 3 above)
Comment (Tougee) [reply app/src/main/java/one/mixin/android/ui/conversation/FriendsFragment.kt]: fixed

**Pull request merged**